docs/_pages/releases.md

```diff
@@ -13,36 +13,36 @@ last_modified_at: 2019-12-30T15:59:57-04:00
-If you are using this feature, you need to relocate the avro dependencies in your custom record payload class to be consistent with internal hudi shading. You need to add the following relocation when shading the package containing the record payload implementation.
-
- ```xml
-<relocation>
-    <pattern>org.apache.avro.</pattern>
-    <shadedPattern>org.apache.hudi.org.apache.avro.</shadedPattern>
-</relocation>
- ```
+ * Dependency Version Upgrades
+   - Upgrade from Spark 2.1.0 to Spark 2.4.4
+   - Upgrade from Avro 1.7.7 to Avro 1.8.2
+   - Upgrade from Parquet 1.8.1 to Parquet 1.10.1
+   - Upgrade from Kafka 0.8.2.1 to Kafka 2.0.0 as a result of updating spark-streaming-kafka artifact from 0.8_2.11/2.12 to 0.10_2.11/2.12.
+ * **IMPORTANT** This version requires your runtime spark version to be upgraded to 2.4+.
+ * Hudi now supports both Scala 2.11 and Scala 2.12, please refer to [Build with Scala 2.12](https://github.com/apache/incubator-hudi#build-with-scala-212) to build with Scala 2.12.
+   Also, the packages hudi-spark, hudi-utilities, hudi-spark-bundle and hudi-utilities-bundle are changed correspondingly to hudi-spark_{scala_version}, hudi-spark_{scala_version}, hudi-utilities_{scala_version}, hudi-spark-bundle_{scala_version} and hudi-utilities-bundle_{scala_version}.
+   Note that scala_version here is one of (2.11, 2.12).
+ * With 0.5.1, we added functionality to stop using renames for Hudi timeline metadata operations. This feature is automatically enabled for newly created Hudi tables. For existing tables, this feature is turned off by default. Please read this [section](https://hudi.apache.org/docs/deployment.html#upgrading), before enabling this feature for existing hudi tables.
+   To enable the new hudi timeline layout which avoids renames, use the write config "hoodie.timeline.layout.version=1". Alternatively, you can use "repair overwrite-hoodie-props" to append the line "hoodie.timeline.layout.version=1" to hoodie.properties. Note that in any case, upgrade hudi readers (query engines) first with 0.5.1-incubating release before upgrading writer.
+ * CLI supports `repair overwrite-hoodie-props` to overwrite the table's hoodie.properties with specified file, for one-time updates to table name or even enabling the new timeline layout above. Note that few queries may temporarily fail while the overwrite happens (few milliseconds).
+ * DeltaStreamer CLI parameter for capturing table type is changed from --storage-type to --table-type. Refer to [wiki](https://cwiki.apache.org/confluence/display/HUDI/Design+And+Architecture) with more latest terminologies.
```

Comment:
Would you please also change to `--storage-type` , `--table-type`, since I find --storage-type and --table-type are changed to -storage-type and -table-type after regenerated in the site(https://hudi.apache.org/releases.html), thanks.

Reply:
Done